source/community.html.erb

```diff
@@ -2,12 +2,66 @@
+<a href="/blog">Hanami Blog</a>
+
+<h2 class="page-header">Chat Room</h2>
+<p>The Gitter room is the official chat room for Hanami. You can drop a quick question here to try and get it answered quickly, ask for guidance, or engage on a short discussion on a feature or project you are interested in. Make sure to keep the discussion organised, and consider using the forum for longer or more involved discussions.</p>
+<a href="https://gitter.im/hanami/chat" target="_blank">hanami/chat</a>
+
+<h2 class="page-header">Twitter</h2>
+<p>The Twitter account will often tweet general announcements on releases and events, and retweet any interesting articles from the tweetosphere. Be sure to mention @hanamirb if you are tweeting about Hanami!</p>
+<a href="https://twitter.com/hanamirb" target="_blank">@hanamirb</a>
+
+<h2 class="page-header">Reddit</h2>
+<p>The Hanami subreddit contains announcements on new releases, interesting gems, blog posts, and is in general a good place to share your work with the rest of the Hanami community.</p>
+<a href="https://www.reddit.com/r/hanamirb/" target="_blank">r/hanamirb</a>
+
+<h2 class="page-header">StackOverflow</h2>
+<p>There is a hanami tag in StackOverflow. Ask for help with your Hanami project there, or help other people solve their problems - learning &amp; sharing all the way through.</p>
+<a href="https://stackoverflow.com/questions/tagged/hanami" target="_blank">hanami tag</a>
+
+<h2 class="page-header">Awesome Hanami</h2>
+<p>Following the awesome-lists tradition, the awesome-hanami Github repository contains a curated list of Hanami projects, gems, tools and tutorials.</p>
+<a href="http://awesome-hanami.org" target="_blank">awesome-hanami.org</a>
+
+<h2 class="page-header">Mailing List</h2>
+<p>To stay updated with the latest releases, to receive code examples, implementation details and announcements, please consider to subscribe to the <a href="/mailing-list">Hanami mailing list</a>.</p>
+
+<h2 class="page-header">Local User Groups</h2>
+<p>Find and join a local Hanami user group and spread the word about Hanami. Share experiences and help others to get started!</p>
+<ul>
+  <li><a href="http://www.meetup.com/hanamirb-sp-brazil-meetups" target="_blank">Hanami User Group @ São Paulo, Brazil</a> (<a href="https://twitter.com/hanamirb_sp" target="_blank">@hanamirb_sp</a>)</li>
+</ul>
+
+<h2 class="page-header">Contribute to Hanami</h2>
+<p>Help is welcome and much appreciated, whether you are an experienced developer or just looking for sending your first pull request. Please check the open tikets.</p>
+<a href="https://github.com/search?utf8=%E2%9C%93&q=user%3Ahanami+state%3Aopen+label%3Aeasy+label%3Ahelp-wanted&type=Issues&ref=searchresults" target="_blank">Open Tickets</a>
+
+<h2 class="page-header">Images</h2>
+<p>Hanami is fully open source. That's why you can download our logo in different formats:</p>
```

Comment:
I'd rephrase it with:

"Hanami logo it's &copy; Luca Guidi, but you can download and use it for non-commercial scopes."